canonical-web-and-design/jp.ubuntu.com
# Update and Switch to github actions

## Done

Complete set of workflows in Github actions

## QA

Github actions checks should to turn green:
lint-js
lint-python
lint-scss
run-dotrun
run-image
test-js
test-python
